src/Backend/SgateShopgatePlugin/Bootstrap.php

```diff
@@ -923,6 +923,17 @@ protected function getFormElementStructure(
                     ->setPosition($position++)
             );
 
+        $formElementCheckbox                 = new FormElementCheckbox();
+        $formElementOptionsContainerCheckbox = new FormElementOptionsContainerCheckbox();
+        $formElements[]                      = $formElementCheckbox
+            ->setKey('SGATE_EXPORT_PRODUCT_INACTIVE')
+            ->setOptions(
+                $formElementOptionsContainerCheckbox
+                    ->setLabel('Inaktive Produkte exportieren')
```

Comment:
```suggestion
                    ->setLabel('Inaktive Produkte exportieren')
                    ->setDescription('Inaktive Produkte werden standardm&auml;&szlig;ig nicht exportiert. Wird diese Einstellung auf "Ja" gesetzt, werden sie als "nicht verk&auml;uflich" exportiert.')
```

Reply:
Thank you for the PR, I had a small suggestion adding a description to the new config field. Apart from that, the Postman test might want to clean up the product it creates.

Reply:
@Rekhyt done